recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
```

Comment:
sort of bikeshedding here, but this is pretty wordy. You might consider dropping the `-meta`. `conda install -c billingegroup-meta -c conda-forge` is a lot to type.